### DISCUSSION TITLE
MVS TV [MX]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#446] [Show and tell]
### Channel ID

MVSTV.mx

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://dailymotion.com/video/x8clp0w

### Notes

Geo-blocked

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://dailymotion.com/video/x8clp0w (offline)

**Comment by ghost** [comment]
no stream
